sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileBasedSource.java

```diff
@@ -131,6 +131,12 @@ protected FileBasedSource(
 
     // This field will be unused in this mode.
     this.emptyMatchTreatment = EmptyMatchTreatment.DISALLOW;
+
+    LOG.info(
```

Comment:
Should this be DEBUG? For each source splitted it will emit one log. Could introduce lots of logs in large pipeline.

If set to DEBUG one can see this log when `--sdkHarnessLogLevelOverride` is configured